tests/test_my_project.py

```diff
@@ -27,3 +33,115 @@ def test_python_dot_org(driver: WebDriver):
 
     # Assert that the first search result is some text...
     assert first_result.text == 'PEP 394 -- The "python" Command on Unix-Like Systems'
+
+
+def test_expanding_menu_bar(driver: WebDriver):
+
+    driver.get("https://machinist.flapflap.io/")
```

Comment:
Make this a global variable so that you dont have to declare it in every test function

Reply:
You told me to do that in my tests lol

Reply:
yes, I do have the globar var i think i forgot it for this test